sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,21 +1,21 @@
 ## Release History
 
-## 4.18.0-beta.2 (Unreleased)
-
-
-### 4.18.0-beta.1 (2021-08-11)
+### 4.18.0 (2021-08-16)
 #### New Features
 * Integrated cosmos diagnostics with open telemetry tracer.
-* Added `BulkExecutionOptions` and `BulkExecutionThresholds`.
 
 #### Key Bug Fixes
 * Added reactor netty timeline to query plan calls.
 * Fixed serialization warning on `clientSideRequestDiagnostics`.
-* Fixed an issue when `IdleEndpointTimeout` is 0.
+* Fixed an issue when `IdleEndpointTimeout` is set to 0 in `DirectConnectionConfig`.
 * Added retry for `PrematureCloseException`.
-* Fixed a hang issue in bulk executor.
+* Fixed an issue where application hangs in bulk executor.
 * Fixed an issue which preventing recovery from 410/0 after split.
-* Added TransportRequestChannelAcquisitionContext in cosmos diagnostics.
+* Added `TransportRequestChannelAcquisitionContext` in `CosmosDiagnostics`.
```

Comment:
should we remove it since it is disabled by default?

Reply:
Done. `BulkExecutionOptions` is still Beta, so didn't include it.